wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallGridView/NetworkConditionIndicatorView.swift

```diff
@@ -46,7 +46,7 @@ final class NetworkConditionIndicatorView: UIView, RoundedViewProtocol {
             label.leadingAnchor.constraint(equalTo: leadingAnchor, constant: 10),
             label.trailingAnchor.constraint(equalTo: trailingAnchor, constant: -10)
         ])
-        backgroundColor = UIColor.nameColor(for: .brightOrange, variant: .light)
+        backgroundColor = overrideUserInterfaceStyle == .dark ? UIColor.accentDarken : UIColor.accent()
```

Comment:
The `accent color` is a `semantic` color. We should not check interfaceStyle manually.

Reply:
Why was the color always orange before? Are you sure we should use an accent color?

Reply:
I checked the designs, it should be accent color, yes.